packages/actor-init-query/webpack.config.ts

```diff
@@ -22,9 +21,6 @@ function createConfig(packagePath: string): Configuration {
       libraryTarget: 'var',
       library: 'Comunica',
     },
-    plugins: [
-      new ProgressPlugin(),
```

Comment:
Could we keep this one?

Reply:
Yes, if I figure out why the import keeps failing after moving the config back to the subpackage. It even failed when I added the webpack dependency into the subpackage.

Reply:
Adjusting the import fixed it, so it was probably just some ESM vs CJS thing. This is restored now.